wasmcloud-test-util/src/provider_test.rs

```diff
@@ -76,10 +181,10 @@ pub struct ProviderProcess {
     pub actor_id: String,
     pub path: PathBuf,
     pub proc: std::process::Child,
-    pub config: TomlMap,
+    pub config: Config,
     pub nats_client: async_nats::Client,
     pub rpc_client: RpcClient,
-    pub timeout_ms: std::sync::Mutex<u64>,
+    pub timeout_ms: std::sync::Mutex<u128>,
```

Comment:
do we really need to switch from a maximum millis of `18_446_744_073_709_551_615` to `340_282_366_920_938_463_463_374_607_431_768_211_455` milliseconds?

Reply:
That's purely to avoid the cast on L334 https://github.com/wasmCloud/wasmcloud-test/pull/22/files#diff-c75b57f6e0342f1067b65574c818714e0e4a2c728556c8555a9999bd80d11fdfR334, because [`as_milis` returns `u128`](https://doc.rust-lang.org/std/time/struct.Duration.html#method.as_millis)

Reply:
hah. I had no idea as_millis was a u128. 😆